src/init.cpp

```diff
@@ -1118,13 +1118,35 @@ bool AppInit2(boost::thread_group& threadGroup)
     }
 
     if (!fTxIndex) {
-        return InitError(_(
-                "Disabled transaction index detected.\n\n"
-                "Omni Core requires an enabled transaction index. To enable "
-                "transaction indexing, please use the \"-txindex\" option as "
-                "command line argument or add \"txindex=1\" to your client "
-                "configuration file."
-            ));
+        // ask the user if they would like us to modify their config file for them
+        std::string msg = _("Disabled transaction index detected.\n\n"
+                            "Omni Core requires an enabled transaction index. To enable "
+                            "transaction indexing, please use the \"-txindex\" option as "
+                            "command line argument or add \"txindex=1\" to your client "
+                            "configuration file within your data directory.\n\n"
+                            "Configuration file: "); // allow translation of main text body while still allowing differing config file string
```

Comment:
I've never worked with Transifex, but I think words and texts to translate have to be isolated, such as `"File"` vs. `"File: "`. A bit cumbersome though, hehe. Another example:

```c++
std::string msg = _("Disabled transaction index detected.") + "\n\n";
msg += _("Omni Core requires an enabled transaction index. To enable "
         "transaction indexing, please use the \"-txindex\" option as "
         "command line argument or add \"txindex=1\" to your client "
         "configuration file within your data directory.") + "\n\n";
msg += _("Configuration file") + ": ";
```